### DISCUSSION TITLE
Connected App 6.4.1 does not work anymore

### DISCUSSION THREAD

**Discussion by donsaimon** [BimmerGestalt/AAIdrive#369] [General]
Since a couple of days, Connected App 6.4.1 does not work anymore... I can not log in...
Someone else with this problem?

**Comment by hufman** [comment]
I understand that BMW Connected shut down in June, and blocking logins is how I suspected they would do it. It should still run the car connection though, which is why I added the Car Information tab to replace the info that would otherwise be visible in BMW Connected.

**Reply by donsaimon** [reply]
I only noticed this thin a couple of days ago, when I accidentally logged out. Then I could not log in again... I will use My BMW app. Is it compatible with your app without issues?

**Reply by hufman** [reply]
Yes the recent 1.3 release (and any later build) is fully compatible with MyBMW.

**Reply by donsaimon** [reply]
I currently use 1.3rc2-13-c87b487 build (21st June) that I found very very very stable (at least with Connected App). Is that build OK even with My BMW app?

**Reply by hufman** [reply]
Yes the RCs have MyBMW support too.

**Reply by donsaimon** [reply]
Shall I expect 100% of connections success as it was with Connected App?

**Reply by hufman** [reply]
My experience has been 100% reliable (except when the MyBMW app logs out every month or so), but everyone's phone is different and there might need to be Power Saving or Background Execution settings to configure that you may have already applied to BMW Connected.

**Reply by donsaimon** [reply]
Ok thanks a lot!
